### DISCUSSION TITLE
The imported password does not match any known hash format

### DISCUSSION THREAD

**Discussion by amirbrzz1** [ory/kratos#2781] [Q&A]
Hi Guys 
I got this error too by the below payload

     credentials: {
            password: {
                   config: {
                       hashed_password: "2710.ANlSrEDPfo+IxpN24Q6eKobjDgpigKPnixLSabVzRzWc348ul9GkoNwCKQv6PIWw6g==",
                     },
              },
       },
All of the hashed passwords starts with 2710.A...
like :
2710.ANlSrEDPfo+IxpN24Q6eKobjDgpigKPnixLSabVzRzWc348ul9GkoNwCKQv6PIWw6g==
2710.AGUkLKG7mb431uOReeR16u6VreIfYKBKu/tsoNW9w+OtcpalhTt/BuGKYkZpoQAoYw==
2710.AEb5ZOc81Hyt4X2iq73yjZGpfpnBojU+y6fRq+TpN5EBUB5OQTemKqjE9UHzIdEbXA==
2710.AF7fxxNtdf89YVMdvM20HXbErZbOr8ewYuB/b/xWntxbY/DElxOo5R8EqzBdyEy3Dw==
2710.ACIWUs28BhPZhOOhTtCzRoopscQXT/px/aRQndHRmwH2bWnaned/eb3r6OAmmV5YBQ==

But I can't figure out this format, represents which hash algorithm
I have an old asp.net application that uses the BrockAllen.MembershipReboot library to manage identities
Can you please give me a clue?

**Comment by vinckr** [comment]
Hey @amirbrzz1 
The hashed passwords come from Ory Kratos or from somewhere else?

**Comment by amirbrzz1** [comment]
Hey @vinckr 
these hashed passwords belong to an old program in .Net that hashed by PKBDF2.
I want to transfer my identities with their password to Ory Kratos
should I change the kratos.yaml  file like this :

hashers:
    algorithm: pbkdf2
    pbkdf2:
         iterations: 2710
